compiler/src/dmd/backend/machobj.d

```diff
@@ -191,7 +191,7 @@ int mach_seg_data_isCode(const ref seg_data sd)
 
 __gshared
 {
-extern (C++) extern Rarray!(seg_data*) SegData;
+import dmd.backend.code: SegData;
```

Comment:
move this import out of the `__gshared` block

Reply:
Done